sources/stylesheets/content-area/_text.scss

```diff
@@ -330,6 +331,12 @@
     }
   }
 
+  .readmore-link {
```

Comment:
The selector and it rules should be defined in the [article block stylesheet partial](https://github.com/Voog/design-santiago/blob/master/sources/stylesheets/components/articles/_article.scss).